raven_framework

```diff
@@ -11,5 +11,54 @@ SCRIPT_DIR=`(cd $SCRIPT_DIRNAME; pwd)`
 
 source $SCRIPT_DIR/scripts/setup_raven_libs
 
-python $SCRIPT_DIR/framework/Driver.py "$@"
+# set up run keywords
+# "ARGS" stores command line arguments not treated in this file (passed through)
+declare -a ARGS
+# by default run in "opt" mode
+MODE='opt'
+# loop through arguments and look for those that are handled before Python is called
+for var in "$@"; do
+  if [ "$var" = '-D' ]; then
+    # run in development mode (assertions not stripped)
+    MODE='dev'
+    continue
+  elif [ "$var" = 'help' ]; then
+    # display "help" print
+    MODE='help'
+    continue
+  fi
+  # otherwise, pass through arguments
+  ARGS[${#ARGS[@]}]="$var"
+done
+
+# run Driver.py based on the mode chosen
+if [ "$MODE" = 'help' ]; then
```

Comment:
I think we should set $MODE = 'help' if $@ is empty as well

Reply:
Done.